rcl/src/rcl/expand_topic_name.c

```diff
@@ -71,14 +71,7 @@ rcl_expand_topic_name(
   rmw_ret = rmw_validate_node_name(node_name, &validation_result, NULL);
   if (rmw_ret != RMW_RET_OK) {
     RCL_SET_ERROR_MSG(rmw_get_error_string().str);
-    switch (rmw_ret) {
-      case RMW_RET_INVALID_ARGUMENT:
-        return RCL_RET_INVALID_ARGUMENT;
-      case RMW_RET_ERROR:
-      // fall through on purpose
-      default:
-        return RCL_RET_ERROR;
-    }
+    return rcl_convert_rmw_ret_to_rcl_ret(rmw_ret);
```

Comment:
I think this adds a couple of potential return codes that didn't exist previously (looking at `rcl_convert_rmw_ret_to_rcl_ret`. I think the header needs to be updated.

Reply:
Hmm, looking at `rmw_validate_node_name` and `rmw_validate_namespace` documentation, these can only return `RMW_RET_OK`, `RMW_RET_ERROR`, and `RMW_RET_INVALID_ARGUMENT`. Aren't we covered already?

Reply:
Good point. It looks fine then.